protos/google/cloud/speech/v1/cloud_speech.proto

```diff
@@ -366,14 +366,6 @@ message SpeakerDiarizationConfig {
   // flexibility by allowing the system to automatically determine the correct
   // number of speakers. If not set, the default value is 6.
   int32 max_speaker_count = 3;
-
```

Comment:
This comes from a proto change:
https://github.com/googleapis/googleapis/commit/4e952e7e2bb0dd2ef389d552d48f44c8dc4b5f8f#diff-393982a1acc89cf84356ac06ca833840